p/scripts/extra.js

```diff
@@ -98,28 +98,40 @@ function init_crypto_form() {
 }
 // </crypto form (Web login)>
 
-function showPW(ev) {
-	if (ev.buttons || ev.key == ' ' || ev.key.toUpperCase() == 'ENTER') {
-		const passwordField = document.getElementById(this.getAttribute('data-toggle'));
-		passwordField.setAttribute('type', 'text');
-		this.classList.add('active');
+let timeoutHide;
+
+function showPW_this(ev) {
+	const id_passwordField = this.getAttribute('data-toggle');
+	if (this.classList.contains('active')) {
+		hidePW(id_passwordField);
+	} else {
+		if (ev.type == 'click' || ev.buttons || ev.key == ' ' || ev.key.toUpperCase() == 'ENTER') {
```

Comment:
```suggestion
		if (ev.type === 'click' || ev.buttons || ev.key === ' ' || ev.key.toUpperCase() === 'ENTER') {
```

(I know this line has just been moved, but let's address this tiny aspect at the same time)